Release-Notes.adoc

```diff
@@ -1,74 +1,82 @@
-* Pods no longer random fail to start occasionally 
-* Includes these CF component versions 
-** diego-release: 1.35 
-** cf-mysql-release: 36.10.0 
-** cflinuxfs2-release: 1.187.0 
-** routing-release: 0.172.0 
-** garden-runc-release: 1.11.1 
-** nats-release: 22 
-** capi-release: 1.49.0 
-* Includes these CF buildpack versions 
-** go-buildpack-release: 1.7.19-16-g37cc6b4 
-** binary-buildpack-release: 1.0.17 
-** nodejs-buildpack-release: 1.5.30-13-g584d686 
-** ruby-buildpack-release: 9adff61 
-** php-buildpack-release: ea8acd0 
-** python-buildpack-release: 1.5.16-14-ga2bbb4c 
-** staticfile-buildpack-release: 1.4.0-12-gdfc6c09 
-** dotnet-core-buildpack-release: 1.0.26-14-gf951834 
-** java-buildpack-release: 3.16-18-gfeab2b6 
+= Release Notes - SUSE Cloud Application Platform
+
+== 1.1.0 Release April 2018
+
+=== What's new 
+
+* Now supported on Microsoft Azure Container Services (AKS)
+* Cloud Foundry component and buildpack updates (see <<features-and-fixes,Features and Fixes>>)
+* PostgreSQL and MySQL service broker sidecars, configured and deployed via Helm
+* +cf backup+ CLI plugin for saving, restoring, or migrating CF data and applications 
+
+See the link:https://www.suse.com/documentation/cloud-application-platform-1/[Deployment Guide] for details information how to deploy Cloud Application Platform.
+
+=== Configuration Changes
+
+IMPORTANT: Changes to the format of `values.yaml` for SCF and UAA require special handling when upgrading from 1.0 to 1.1 if you are reusing configuration files (e.g. `scf-config-values.yaml`):
+
+** Several values formerly set under `env:` are now set under `secrets:`. Any `_PASSWORD`, `_SECRET`, `_CERT`, or `_KEY` value explicitly set in `values.yaml` for CAP 1.0 should be moved into the `secrets:` section before running `helm upgrade` with the revised `values.yaml` 
+
+** **These secrets must be resupplied on each upgrade** (e.g. the `CLUSTER_ADMIN_PASSWORD`, `UAA_ADMIN_CLIENT_SECRET`) as they will not be carried forward automatically.
+
+** To rotate secrets, increment the `kube.secret_generation_counter` (immutable generated secrets will not be reset).
+
+** The `kube.external_ip` variable has been changed to `kube.external_ips`, allowing for services to be exposed on multiple Kubernetes worker nodes (e.g. behind a TCP load balancer). Before upgrading, change the setting or add a new setting specified as an array. For example:
+
+  kube.external_ip=10.1.1.1
+  kube.external_ips=["10.1.1.1"]
+
+** Both variables can exist at the same time and be set to the same value for those in mixed version environments. To specify multiple addresses:
+
+  kube.external_ips=["1.1.1.1", "2.2.2.2"]
+
+
+=== Known Issues
+
+* Do not set the `mysql` or `diego_access` roles to more than one instance each. Doing so can cause problems with subsequent upgrades which could lead to loss of data. Scalability of these roles will be enabled in an upcoming maintenance release.
```

Comment:
Same for `routing-api`. 
I didn't know `mysql` could not be scaled! What's the reason behind it?